test/functional/ui/screen_basic_spec.lua

```diff
@@ -951,6 +951,46 @@ local function screen_tests(linegrid)
                                                            |
     ]])
   end)
+
+  describe('more prompt', function()
```

Comment:
I don't think this belong in `screen_basic_spec.lua`. Perhaps a "pager" section of `ui/messages_spec.lua` . There will be more pager tests with #10930 .

Reply:
Ok, moved.